core/textile/event_handler.go

```diff
@@ -27,3 +28,28 @@ func (h *defaultListenerHandler) OnRemove(bucketData *bucket.BucketData, listenE
 func (h *defaultListenerHandler) OnSave(bucketData *bucket.BucketData, listenEvent *tc.ListenEvent) {
 	log.Info("Default Listener Handler: OnSave")
 }
+
+type restorerListenerHandler struct {
+	synchronizer sync.Synchronizer
+}
+
+func newRestorerListenerHandler(synchronizer sync.Synchronizer) *restorerListenerHandler {
+	return &restorerListenerHandler{
+		synchronizer: synchronizer,
+	}
+}
+
+func (h *restorerListenerHandler) OnCreate(bucketData *bucket.BucketData, listenEvent *tc.ListenEvent) {
```

Comment:
@dmerrill6 

Do you think it's possible to make it more granular in the next release?

Reply:
I think it's possible, although I think most of the events trigger OnSave (think OnCreate and OnDelete are triggered on bucket creation/deletion and not at file level)